backend/routes/auth/tokens.js

```diff
@@ -13,9 +13,9 @@
     try {
       const userIP = req.ip;
       // Create a token with user IP
-      // console.log(process.env.SECRET_KET);
-      let key = process.env.SECRET_KET;
-      let expiresIn = 60 * 60 * 24; // 24 hours in seconds
```

Comment:
Please put back that comment it's important

Reply:
OK

Reply:
Done